docs/assets/diagrams/linking/06-sign-challenge.puml

```diff
@@ -0,0 +1,125 @@
+@startuml
+
+title: Sign Challenge
+
+hide footbox
+
+box "Mobile Device" #Light
+  actor Alice
+  participant "PISP App" as App
+end box
+box "PISP" #Light
+  participant "Firebase" as Firebase
+  participant "PISP Server" as Server
+end box
+box "Mojaloop" #Light
+  participant "Third-party\nAPI Adapter" as Adapter
+end box
+
+'start flow
+
+App -> Server ++: ""**PUT /consents/{ID}**""\n\
+""{""\n\
+""  "requestId": string,""\n\
+""  "initiatorId": string,""\n\
+""  "participantId": string,""\n\
+""  "scopes": [""\n\
+""    {""\n\
+""      "accountId": string,""\n\
+""      "actions": [string]""\n\
+""    }""\n\
+""  ],""\n\
+""  "credential": {""\n\
+""    "id": string,""\n\
+""    "credentialType": enum, // required?""\n\
+""    "status": enum,""\n\
+""    "challenge": {""\n\
+""      "payload": string,    // base64""\n\
+""      "signature": string   // base64""\n\
+""    },""\n\
+""    "payload": string       // base64""\n\
+""  }""\n\
+""}""
+
+App <-- Server: ""**HTTP 202** (Accepted)   // not 200?""
```

Comment:
I think it is `202` because the App will still be expecting to hear something back in the future regarding this resource.

Reply:
Oh okay, it just seemed a bit weird as in almost all Mojaloop APIs `PUT` requests typically have `200 OK` response.

Reply:
Indeed you are right. We brought this up with JJ before, and this was a `202` because there is still more work to be done, and the client should still be waiting for a callback.

Reply:
okay, got it